docuilib/src/components/pageComponents/Usage.tsx

```diff
@@ -2,14 +2,18 @@ import _ from 'lodash';
 import React from 'react';
 import CodeBlock from '@theme/CodeBlock';
 import '../ComponentPage.module.scss';
+import ReactLiveScope from '../../theme/ReactLiveScope';
+import UILivePreview from '../UILivePreview';
 
 export const Usage = ({component}) => {
+  const componentLivePlaygroundSupport = !!ReactLiveScope[component.name];
+  const code = component.snippet?.map(item => _.replace(item, new RegExp(/\$[1-9]/, 'g'), '')).join('\n');
   if (component.snippet) {
-    return (
+    return componentLivePlaygroundSupport ? (
```

Comment:
You can simplify: `return componentLivePlaygroundSupport ? <UILivePreview code={code}/> : <CodeBlock language="jsx">{code}</CodeBlock>;`